daml-lf/validation/src/main/scala/com/digitalasset/daml/lf/validation/Typing.scala

```diff
@@ -986,7 +986,7 @@ private[validation] object Typing {
         typeOfExerciseInterface(tpl, choice, cid, arg, guard)
       case UpdateExerciseByKey(tpl, choice, key, arg) =>
         typeOfExerciseByKey(tpl, choice, key, arg)
-      case UpdateFetch(tpl, cid) =>
+      case UpdateFetchTemplate(tpl, cid) =>
         typeOfFetch(tpl, cid)
```

Comment:
while we're at it, maybe we should rename `s/typeOfFetch/typeOfFetchTemplate/g` too